src/main/java/org/prebid/server/bidder/gumgum/GumgumBidder.java

```diff
@@ -127,6 +135,13 @@ private static Imp modifyImp(Imp imp) {
         return imp;
     }
 
+    private void validateVideoParams(Video video) {
+        if (video.getW() == 0 || video.getH() == 0 || video.getMinduration() == 0 || video.getMaxduration() == 0
```

Comment:
What is happen if the field will be null?
Seems we need to fix the case for all mentioned fields here.